scripts/travis/build.sh

```diff
@@ -98,6 +98,11 @@ fi
 # Build the current project
 mvn -B -ntp -V -q install -DskipTests -Dmaven.javadoc.skip=true -Pbuild-docker-images -Pags ${REPO_IMAGE} ${SHARE_IMAGE}
 
+#Build alfresco image with jdbc drivers
+MYSQL_TAG=$(mvn help:evaluate -Dexpression=dependency.mysql.version -q -DforceStdout)
```

Comment:
Does it make sense to add other drivers in this PR?

Reply:
It may. But I assumed that better is to add one database in all places in one PR to prevent missing some parts.